ansible/vars.yml

```diff
@@ -0,0 +1,21 @@
+workspace: /var/www/AcmeProject
+
+timezone: Europe/Paris
```

Comment:
I would not set that at all, as the default is `UTC` which fit best for an open source project IMHO